mod/missions/actions/missions/mission-offer.php

```diff
@@ -51,10 +51,12 @@
 					'href' => elgg_get_site_url() . 'missions/view/' . $mission->guid,
 					'text' => elgg_echo('missions:respond')
 			));
-			 
-			$subject = elgg_echo('missions:offers_you_a_spot', array(elgg_get_excerpt($mission->job_title, elgg_get_plugin_setting('mission_job_title_card_cutoff', 'missions'))), $applicant->language);
-			$body = elgg_echo('missions:offers_you_a_spot_more', array(elgg_get_excerpt($mission->job_title, elgg_get_plugin_setting('mission_job_title_card_cutoff', 'missions'))), $applicant->language) . " {$finalize_link}" . '.';
-			mm_notify_user($applicant->guid, $mission->guid, $subject, $body);
+			
+			$subject = elgg_echo('missions:offers_you_a_spot', array(elgg_get_excerpt($mission->job_title, elgg_get_plugin_setting('mission_job_title_card_cutoff', 'missions'))),'en', $applicant->language) .' | '.elgg_echo('missions:offers_you_a_spot', array(elgg_get_excerpt($mission->job_title, elgg_get_plugin_setting('mission_job_title_card_cutoff', 'missions'))),'fr', $applicant->language);
+			$offer_en = elgg_echo('missions:offers_you_a_spot_more', array(elgg_get_excerpt($mission->job_title, elgg_get_plugin_setting('mission_job_title_card_cutoff', 'missions'))),'en', $applicant->language) . " {$finalize_link}" . '.';
+			$offer_fr = elgg_echo('missions:offers_you_a_spot_more', array(elgg_get_excerpt($mission->job_title, elgg_get_plugin_setting('mission_job_title_card_cutoff', 'missions'))),'fr', $applicant->language) . " {$finalize_link}" . '.';
```

Comment:
$applicant->language isn't needed anymore / can be cleaned out of there now that you're specifying en/fr.